tools/gulptasks/dist-examples.js

```diff
@@ -204,12 +205,16 @@ async function createExamples(title, sourcePath, targetPath, template) {
 
     LogLib.success('Created', targetPath);
 
-    const localsidebar = getLocalSidebar(sourcePath.replace(/samples\//, '').replace(/\/demo/, ''));
+    const localsidebar = getLocalSidebar(
+        sourcePath
+            .replaceAll('samples/', '')
+            .replaceAll('/demo', '')
+    );
 
     LogLib.success('Created', targetPath);
     const indexContent = localsidebar
-        .replace(/style=\"display:none;\"/g, '') // remove hidden style
-        .replace(/(?!href= ")(\.\/.+?)(?=")/g, 'examples\/$1\/index.html'); // replace links
+        .replaceAll('style="display:none;"', '') // remove hidden style
```

Comment:
Is it not saver to use regexp and set style as optional?

Reply:
Should be equivalent in this case. For context, this is the kind of file being replaced: https://www.highcharts.com/demo/sidebar.html